### PR TITLE
chore: removed all refs to polygon-parts (MAPCO-3980)

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

File: src/update/metadataMerger.ts [hunk 1]
```diff
@@ -46,7 +42,7 @@ export class MetadataMerger {
       classification: isSwap ? updateMetadata.classification : this.mergeClassification(oldMetadata.classification, updateMetadata.classification),
     };
     newMetadata.productBoundingBox = createBBoxString(newMetadata.footprint as Footprint);
-    newMetadata.sensors = this.polygonPartsToSensors(newMetadata.layerPolygonParts as FeatureCollection);
+    newMetadata.sensors = updateMetadata.sensors;
```
Comment (asafMasa) [inline src/update/metadataMerger.ts]: isSwap - replace
!isSwap - call "mergeUniqueArrays" method

File: src/update/metadataMerger.ts [hunk 2]
```diff
@@ -46,7 +42,7 @@ export class MetadataMerger {
...
     return newMetadata;
```
Comment (asafMasa) [inline src/update/metadataMerger.ts:46]: fix to test each case (use existing tests)

**Review by asafMasa — CHANGES_REQUESTED** [review]

**Review by asafMasa — APPROVED** [review]

**Review by asafMasa — APPROVED** [review]

File: src/layers/models/layersManager.ts [hunk 3]
```diff
@@ -614,8 +593,5 @@ export class LayersManager {
     data.metadata.srsId = data.metadata.srsId === undefined ? '4326' : data.metadata.srsId;
     data.metadata.srsName = data.metadata.srsName === undefined ? 'WGS84GEO' : data.metadata.srsName;
     data.metadata.productBoundingBox = createBBoxString(data.metadata.footprint as GeoJSON);
-    if (!data.metadata.layerPolygonParts) {
```
Comment (asafMasa) [inline src/layers/models/layersManager.ts:617]: set as undefined

**Review by asafMasa — CHANGES_REQUESTED** [review]

**Review by asafMasa — APPROVED** [review]

File: src/update/metadataMerger.ts (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply src/update/metadataMerger.ts]: fixed

File: src/update/metadataMerger.ts (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply src/update/metadataMerger.ts:46]: fixed

Comment (CL-SHLOMIKONCHA) [reply src/update/metadataMerger.ts:46]: fixed

File: src/update/metadataMerger.ts (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply src/update/metadataMerger.ts]: fixed

File: src/layers/models/layersManager.ts (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply src/layers/models/layersManager.ts:617]: fixed

**Pull request merged**